cmd/cloud/deployment.go

```diff
@@ -171,11 +173,14 @@ func newDeploymentCreateCmd(out io.Writer) *cobra.Command {
 			fmt.Println(err)
 		}
 		cmd.Flags().StringVarP(&deploymentType, "type", "", standard, "The Type to use for the Deployment. Possible values can be standard or dedicated.")
+		cmd.Flags().StringVarP(&defaultTaskPodCPU, "default-task-pod-cpu", "", "", "The default task pod CPU to use for the Deployment. Example value: 0.25")
+		cmd.Flags().StringVarP(&defaultTaskPodMemory, "default-task-pod-memory", "", "", "The default taks pod memory to use for the Deployment. Example value: 0.5Gi")
+		cmd.Flags().StringVarP(&resourceQuotaCPU, "resource-quota-cpu", "", "", "The Deployment's CPU resource quota. Example value: 10")
```

Comment:
Does the example value needs to have the unit? Also, do we need to provide defaults for these flags?

Reply:
The defaults come from the CORE API if "" is passed. For some reason cpu does not have units and memory does. Just following what is passed to the CORE API. Got these examples from the CORE API docs and they work